stable/polaris/README.md.gotmpl

```diff
@@ -0,0 +1,29 @@
+# Polaris
+
+[Polaris](https://github.com/FairwindsOps/polaris)
+is a tool for auditing and enforcing best practices in Kubernetes.
+
+## Installation
+We recommend installing polaris in its own namespace.
+
+### Dashboard
+```
+helm repo add fairwinds-stable https://charts.fairwinds.com/stable
+helm install polaris fairwinds-stable/polaris --namespace polaris
+```
+
+### Webhook
+
+A valid TLS certificate is required for the Polaris Validating Webhook. If you have cert-manager installed in your cluster then the helm install below will work. If you don't use cert-manager and don't want to install it then you can supply a CA Bundle with the `webhook.caBundle` parameter and create a TLS secret and pass the name of that secret as `webhook.secretName`
```

Comment:
can we bullet this out a bit? i.e.
```md
If you don't use cert-manager, you'll need to:
* Supply a CA Bundle using the `webhook.caBundle` parameter
* create a TLS secret in your cluster
* pass in the name of that TLS secret using the `webhook.secretName` parameter
```

Also, is there a particular version of cert-manager we need?

Reply:
0.11 through 1.0 should all work just fine. At some point we should bump the API version of the objects we're using.